ui/appui-react/src/appui-react/utils/Placement.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+type Side = "Left" | "Top" | "Right" | "Bottom";
+type Corner = "TopLeft" | "TopRight" | "BottomLeft" | "BottomRight" | "RightTop" | "LeftTop"
+
+/**
+ * Identifies the placement of a given element.
+ * @public
+ */
+export type Placement = Side | Corner;
+
+/**
+ * Copy of appui-abstract's RelativePosition for use in below utilities.
+ * Should be removed N+1 major versions after the deprecation.
+ */
+enum RelativePosition {
```

Comment:
Why not simply import it instead?

Reply:
Eh, figured it would be one less abstract import to clean up, but if we have to clean this up anyway... kind of useless.

Reply:
fixed